### PR TITLE
APcattle zkthon. Challenge 1 and 2 

### PR DESCRIPTION
**Pull request by APcattle**
## Description

This pull request is for the zkEVM Challenge 1,2. 
I have reviewed the zkEVM documentation and shared my understanding through a Twitter thread using the hashtags #zkThon and #zkEVM. 

The link to my Twitter thread is https://twitter.com/ulita_lovely/status/1640066133637099524?s=20 

## Checklist

- [x] I have created a new file named `solution-1.md, solution-2.md` for this challenge.
- [x] I have added the details of my submission to the [Airtable form](https://airtable.com/shr21z0FfPImZfYBQ).
- [x] I have named my branch `APcattle_zkthon`.

### CODE REVIEW

File: solution-1.md
Comment (megabyte0x) [inline solution-1.md:1]: Great work ser 💪🏻

File: solution-2.md
Comment (megabyte0x) [inline solution-2.md:1]: Please add the contract deployed link. 

Thanks

**Review by megabyte0x — COMMENTED** [review]
Also please format the PR accordingly next time. Currently I am closing this one.

File: solution-1.md
Comment (APcattle) [reply solution-1.md:1]: thanks!

File: solution-2.md
Comment (APcattle) [reply solution-2.md:1]: yep, done. 

am i need to do smthelse to submit solution?

**Pull request closed without merging**